examples/extract_data/src/README.md

```diff
@@ -0,0 +1,26 @@
+# Quick setup
+
+## Prerequisites
+
+- Python 3.10
+- Poetry==1.8.4
+
+## Setup environment
+
+Clone the repository and navigate to this example
+
+`cd examples/extract_data/src`
+
+Install dependencies with poetry
+
+`poetry install`
+
+Initiate the environment
+
+`poetry shell`
+
+## Run the example
+
+Input your Airtop key in the .env file.
+
+Change the variable `TARGET_URL` to your desired target in summarize.py.
```

Comment:
We are missing the command to run the python script under `Run the example`.

Reply:
Added